### PR TITLE
Migrate to Swift 5

### PR DESCRIPTION
**Pull request by Thomas06000**

### CODE REVIEW

File: iOS Example/AppDelegate.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
 
-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```
Comment (LittleWat) [inline iOS Example/AppDelegate.swift]: unnecessary space

**Comment by pdshelley** [comment]
Can we get this merged?

**Comment by rotemsOr** [comment]
Can someone please make this happen? My project is broken.

**Comment by halonsoluis** [comment]
@rotemsOr In the meantime, while these gets some attention, you should be able to use the old version in your updated project by adding in the PodFile:

post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|

            if target.name == 'SwiftSocket'
              config.build_settings['SWIFT_VERSION'] = '4.2'
            end

     end
end

**Comment by rotemsOr** [comment]
> @rotemsOr In the meantime, while these gets some attention, you should be able to use the old version in your updated project by adding in the PodFile:
> 
> post_install do |installer|
> installer.pods_project.targets.each do |target|
> target.build_configurations.each do |config|
> 
> ```
>         if target.name == 'SwiftSocket'
>           config.build_settings['SWIFT_VERSION'] = '4.2'
>         end
> 
>  end
> ```
> 
> end

Thank you, will try!

**Pull request merged**